Dockerfile_Service

```diff
@@ -0,0 +1,25 @@
+FROM nikolaik/python-nodejs:python3.8-nodejs16
+
+RUN apt-get update
+
+# Install serverless
+RUN npm install -g serverless && \
+    npm install -g serverless-offline
```

Comment:
These dependencies are defined in package.json. Why don't we install from it?

Same thing for the `RUN serverless plugin install -n serverless-s3-local` down there.

Reply:
Another thing is that you didn't lock the version of the packages to be installed, which is something we would get by using package.json